packer-windows/buildkite-ami.json

```diff
@@ -0,0 +1,73 @@
+{
+  "variables": {
+    "region": "us-east-1"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "region": "{{user `region`}}",
+      "source_ami_filter": {
+        "filters": {
+          "name": "Windows_Server-2019-English-Full-Containers*",
+          "virtualization-type": "hvm"
+        },
+        "owners": ["amazon"],
+        "most_recent": true
+      },
+      "instance_type": "i3.large",
+      "spot_price": "auto",
+      "spot_price_auto_product": "Windows (Amazon VPC)",
+      "user_data_file":"scripts/ec2-userdata.ps1",
+      "communicator": "winrm",
```

Comment:
I'm _reasonably_ sure that it is now possible to use openssh-win32 and packer with great success. https://operator-error.com/2018/04/16/windows-amis-with-even/

Reply:
This is interesting. I can see value in adding ssh access and enabling the `AuthorizedUsersUrl` stack setting which would automatically configure the `.ssh/authorized_keys` file.

https://github.com/buildkite/elastic-ci-stack-for-aws/blob/master/packer/conf/bin/bk-install-elastic-stack.sh#L127-L137